data/tools/codiga.yml

```diff
@@ -1,22 +1,25 @@
-name: Code Inspector
+name: Codiga
 categories:
   - linter
 tags:
+  - apex
   - c
   - cpp
+  - dockerfile
   - go
   - java
   - javascript
+  - kotlin
   - ruby
   - php
   - python
-  - kotlin
   - typescript
+  - scala
   - ci
 license: proprietary
 types:
   - service
-homepage: "https://www.code-inspector.com"
+homepage: "https://www.codga.io"
```

Comment:
it should be codiga.io, you are missing the letter 'i'.

Reply:
My bad, let me make another pull request with the correct letter. Thanks for flagging!

Reply:
Yeah I went ahead and fixed it when I realized that it was pointing to the wrong location. 😄